test/app/features/authentication/presentation/sign_in_stealth/sign_in_stealth_page_test.dart

```diff
@@ -0,0 +1,178 @@
+          const password = 'P4ssw0rd';
+          when(() => AuthenticationModulesMock.passwordValidator
+              .validate(any(), any())).thenAnswer((_) => success(password));
+          when(
+            () => AuthenticationModulesMock.authenticationRepository
+                .signInWithEmailAndPassword(
+                    emailAddress: any(named: 'emailAddress'),
+                    password: any(named: 'password')),
+          ).thenFailure((_) => ServerFailure());
+
+          await theAppIsRunning(tester, const SignInStealthPage());
+
+          await iEnterIntoPasswordField(tester,
+              text: 'Senha', password: password);
+          await iTapText(tester, text: 'Entrar');
+
+          await iSeeText(
+              'O servidor está com problema neste momento, tente novamente.');
+        },
+      );
+
+      testWidgets(
+        'valid password login into app',
+        (tester) async {
+          const password = 'P4ssw0rd';
+          when(() => AuthenticationModulesMock.passwordValidator
+              .validate(any(), any())).thenAnswer((_) => success(password));
+          when(
+            () => AuthenticationModulesMock.authenticationRepository
+                .signInWithEmailAndPassword(
+                    emailAddress: any(named: 'emailAddress'),
+                    password: any(named: 'password')),
+          ).thenSuccess((_) => FakeSessionEntity());
+          when(() => AppModulesMock.modularNavigator.canPop()).thenReturn(true);
+
+          await theAppIsRunning(tester, const SignInStealthPage());
+
+          await iEnterIntoPasswordField(tester,
+              text: 'Senha', password: password);
+          await iTapText(tester, text: 'Entrar');
+
+          verify(() => AppModulesMock.modularNavigator.canPop()).called(1);
+        },
+      );
+
+      testWidgets(
+        'forward to forget my password',
+        (tester) async {
+          when(() => AppModulesMock.modularNavigator.pushNamed(any()))
+              .thenAnswer((_) => Future.value());
+
+          await theAppIsRunning(tester, const SignInStealthPage());
+          await iTapText(tester, text: 'Esqueci minha senha');
+
+          verify(() => AppModulesMock.modularNavigator
+              .pushNamed('/authentication/reset_password')).called(1);
+        },
+      );
+
+      testWidgets(
+        'forward to changed account',
+        (tester) async {
+          when(() => AppModulesMock.modularNavigator.pushNamed(any()))
+              .thenAnswer((_) => Future.value());
+
+          await theAppIsRunning(tester, const SignInStealthPage());
+          await iTapText(tester, text: 'Acessar outra conta');
+
+          verify(() =>
+                  AppModulesMock.modularNavigator.pushNamed('/authentication'))
+              .called(1);
+        },
+      );
+
+      screenshotTest(
+        'looks as expected',
+        fileName: 'sign_in_stealth_page',
+        pageBuilder: () => const SignInStealthPage(),
+      );
```

Comment:
Em alguns testes está sendo agrupado e em outros não estão:

```suggestion
    group('golden test', () {
      screenshotTest(
        'looks as expected',
        fileName: 'sign_in_stealth_page',
        pageBuilder: () => const SignInStealthPage(),
      );
```

Reply:
Ajustado para estar dentro do `group`